Civi/Api4/Generic/AbstractEntityInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+namespace Civi\Api4\Generic;
+
+/**
+ * This is currently a null interface - it allows picking
+ * up AbstractEntities via the ClassScanner
+ */
+interface AbstractEntityInterface {
```

Comment:
This is a good idea. Maybe call it `Api4EntityInterface` since it's the be-all and end-all (if a class entity doesn't implement this interface, it's not an api4 entity!)

Reply:
Have made it EntityInterface (given its already in Api4 namespace)  :slightly_smiling_face: